seatunnel-e2e/seatunnel-flink-connector-v2-e2e/connector-iotdb-flink-e2e/src/test/resources/iotdb/iotdb_source_to_sink.conf

```diff
@@ -54,9 +54,7 @@ source {
 }
 
 transform {
-    sql {
```

Comment:
You also need to fix the logic in the test case, not just remove

Reply:
OK ,thx ;